frontend/src/lib/i18n/locales/en.json

```diff
@@ -260,7 +260,23 @@ Lexbox is free and [open source](https://github.com/sillsdev/languageforge-lexbo
         "project_now_not_confidential": "Project is now not confidential.",
       }
     },
-    "awaiting_approval": "Awaiting approval"
+    "awaiting_approval": "Awaiting approval",
+    "crdt": {
+      "try_fw_lite": "Try FieldWorks Lite",
+      "try_info": "This will make your project available in [FieldWorks Lite](https://software.sil.org/fieldworks/download/fieldworks-lite/). \
```

Comment:
thanks for moving this into translated text. What do you think about changing this to be an order of steps. EG:

1. Make project available (click "Yes" below)
2. while that's happening install FieldWorks Lite
3. login to lexbox from FieldWorks lite
4. Wait for this process to finish if it's not done
5. Open your project in FieldWorks lite (note if it's not showing up it may need to refresh, you can close and re open the app to refresh your project list)

once they click "Yes" we would show that it's syncing the project, but still show the instructions as it might take a while. Also we could show a fun loading animation since it might take so long.